vignettes/model_definition.Rmd

```diff
@@ -130,17 +130,19 @@ To estimate the uncertainty in the nowcasts, we use past nowcast errors and assu
 
 We describe a method which generates retrospective reporting triangles to replicate what would have been available as of time $t^*=s^*$, where $s^* = t^*-m$ for $m = 1, 2, ... M$ to generate $M$ retrospective reporting triangles.
 
-To generate the set of $M$ reporting triangles, we simply remove the last $m$ rows of the existing reporting triangle (or reporting matrix), to generate $M$ truncated incomplete reporting matrices. These can be generated from a reporting triangle or matrix with the function `truncate_triangles()` which ingests a single reporting triangle (or matrix) and returns a list of `n` truncated (potentially incomplete) reporting matrices containing only observations and missing values if present. We then replace the bottom right triangle of each matrix with NAs, assuming these would not have been observed as of $s^*$, using the function `generate_triangles()` which returns a list of `n` retrospective reporting triangles. The method uses each retrospective reporting triangle to re-estimate a delay distribution using the $N$ preceding rows of the reporting triangle before $s^*$, and recomputes a retrospective nowcast, for $M$ realizations of the retrospective reporting triangle (so $M$ different $s^*$ values).
+To generate the set of $M$ reporting triangles, we work backwards from most recent to oldest, removing the last $m$ rows of the existing reporting triangle, to generate $M$ truncated incomplete reporting matrices. These can be generated from a reporting triangle with the function `truncate_triangles()` which ingests a single reporting triangle and returns a list of `n` truncated reporting matrices containing only observations and missing values if present. We then replace the bottom right triangle of each matrix with NAs, assuming these would not have been observed as of $s^*$, using the function `generate_triangles()` which returns a list of `n` retrospective reporting triangles, again in order from most recent to oldest.
```

Comment:
I thought we got rid of the incomplete language and where using truncated vs true reporting triangle only?

Reply:
Good catch, will go through and remove any references to `incomplete` and matrices that are incorrect in this model definition